examples/spack/Dockerfile

```diff
@@ -1,6 +1,5 @@
-# FIXME: was full; doesn't work after autotools update
-# ch-test-scope: skip
-FROM debian9
+# ch-test-scope: full
+FROM centos:7
```

Comment:
This is the upstream CentOS 7 image. Should it be `centos7` to be ours, or `centos:8` or `centos8`?

Reply:
I suppose our centos 7 and 8 images are a bit bulkier with the added sauce needed for openmpi and rpm building. My initial thought was to use a fresh upstream CentOS image but I don't think I have a strong opinion one way or another.